docs/api/expect.md

```diff
@@ -1405,3 +1405,72 @@ Don't forget to include the ambient declaration file in your `tsconfig.json`.
 :::tip
 If you want to know more, checkout [guide on extending matchers](/guide/extending-matchers).
 :::
+
+## expect.addEqualityTesters
+
+- **Type:** `(tester: Array<Tester>) => void`
+
+- **Version:** Since Vitest 1.0.0
```

Comment:
1. Version is now added using `<Badge>` component
2. 1.0.0 is already released. Plan for this is 1.2.0

Reply:
Added `<Badge>` and changed version number.